public/index.html

```diff
@@ -8,8 +8,79 @@
 </head>
 <body>
     <div class="wrap">
-        <h1 class="test-headline">Slidy</h1>
-        <p class="test-paragraph">Залиш свій слід у світі подорожей</p>
-    </div>
+        <div class="container">
+          <div class="left-section">
+            <div class="nav">
+              <div class="nav-logo">
+                <svg xmlns="http://www.w3.org/2000/svg" width="100" height="100" viewBox="0 0 100 100">
+                  <image xlink:href="D:\footer project\media\logo.svg" width="100" height="100" />
+                </svg>
+              </div>
+              <div class="nav-title">НАВІГАЦІЯ ПО САЙТУ</div>
+              <ul class="nav-list">
+                <li><a href="#">Про нас</a></li>
+                <li><a href="#">Контакти</a></li>
+                <li><a href="#">Як користуватись</a></li>
+                <li><a href="#">Головна</a></li>
+              </ul>
+            </div>
+            <div class="social" class="left-section">
```

Comment:
if you want to specify more than one class to one element write class="one-class the-other-one-class"